### DISCUSSION TITLE
Allow changing of tags on mobile

### DISCUSSION THREAD

**Discussion by Moosems** [community/community#57702] [Mobile] [answered]
I think it would be really awesome for a small button to be added allowing users to edit tags on issues and PR's or adding it to the extras menu (the three dots).

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.115.0 (100405004)
Device: iPhone14,7 (iOS 16.2)
</details>

**Comment by Iron-Ham** [comment, marked as answer]
Hi @Moosems,

I'm happy to report that this feature is already part of GitHub Mobile, but perhaps not where you expected it to be.
<details><summary>Step One: Accessing the Info Menu</summary>
<p>
In order to edit information of a pull request or issue, you have to tap the floating "info" button in the bottom right corner:

<img src="https://github.com/community/community/assets/3388381/3d99963a-22ce-4eae-acfc-b3324bb5f868" width =50% height=50%>

</p>
</details> 

<details><summary>Step Two: Selecting an Edit Field</summary>
<p>

From there, you'll be greeted by a screen which allows you to edit the fields of the issue or pull request, if you have edit permission over the issue or pull.

<img src="https://github.com/community/community/assets/3388381/41c39089-3e0a-46d3-b386-36a28d5a964c" width =50% height=50%>

</p>
</details> 

<details><summary>Step Three: Modifying the field</summary>
<p>

Select the field you're looking to change -- in this case, labels -- and edit the labels as you see fit.

<img src="https://github.com/community/community/assets/3388381/68d68297-55e7-49b2-96a4-159602b460f4" width =50% height=50%>

</p>
</details> 

______


I hope this helps answer your question! 

Thanks,
Hesham
